Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiRequestBus.h

```diff
@@ -95,9 +98,9 @@ namespace AzToolsFramework::ViewportUi
         virtual void RemoveTextField(TextFieldId textFieldId) = 0;
         //! Sets the visibility of the text field.
         virtual void SetTextFieldVisible(TextFieldId textFieldId, bool visible) = 0;
-        //! Create the highlight border for Component Mode.
-        virtual void CreateViewportBorder(const AZStd::string& borderTitle) = 0;
-        //! Remove the highlight border for Component Mode.
+        //! Create the highlight border for editor modes with optional back button to exit the given editor mode.
+        virtual void CreateViewportBorder(const AZStd::string& borderTitle, AZStd::optional<ViewportUIBackButtonCallback> backButtonCallback) = 0;
+        //! Remove the highlight border for editor modes.
```

Comment:
suggestion: Thanks for fixing the comment! 😝 Any chance just having it be `Remove the highlight border` and `Create the highlight border` has it might be used in other contexts in future, not just Editor Modes.

Reply:
no probs, fixed 👍